database/postgres/postgres.go

```diff
@@ -35,6 +35,7 @@ var (
 type Config struct {
 	MigrationsTable string
 	DatabaseName    string
+	SchemaName      string
```

Comment:
I'm not a fan of the schema name being exported, but you're just following the existing pattern where the database name is exported. Let's leave this as is for now and we'll tackle unnecessarily exported field names in a later backwards incompatible version of `migrate`.